### ISSUE TITLE
RCC H735 conflicting rules

### ISSUE THREAD

**Issue by burrbull** [stm32-rs/stm32-rs#605]
Rules in stm32h735 (RM0468) are partially incompatible with that in `common_patches/h7_common_singlecore.yaml`.
```console
svdtools makedeps devices/stm32h735.yaml .deps/stm32h735.d
In String("name"): conflicting rules String("OCTOSPISEL") and String("QSPISEL"), ignored
In String("name"): conflicting rules String("SWPMISEL") and String("SWPSEL"), ignored
In String("name"): conflicting rules String("I2C1235SEL") and String("I2C123SEL"), ignored
In String("name"): conflicting rules String("USART16910SEL") and String("USART16SEL"), ignored
```

**Comment by richardeoin** [comment]
It seems that the correct rules (the ones in `devices/stm32h735.yaml`) are used, but indeed the current situation is ambiguous and should be fixed. I'm a bit confused on a couple of things:

* I think the conflicting rules are in `common_patches/h7_rcc_src_sel.yaml`. Not sure on the connection to `common_patches/h7_common_singlecore.yaml`
* How do I reproduce the log above? Is the released svdtools 0.1.16 enough, or do I need master or some other branch?

**Comment by burrbull** [comment]
I've found this using rust version of `svd patch`: https://github.com/burrbull/svdtools/tree/riir2/src

The problem is `yaml_includes` function which is not smart enough to find duplicates or conflicts in rules.
